### PR TITLE
Github Actions CI

### PR DESCRIPTION
**Pull request by bvitaliyg**
+ [ ] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-android/blob/master/CONTRIBUTING.md)

Remember to include the following changes:
+ [ ] `UNRELEASED.md`
+ [ ] `README.md`
+ [ ] Tests
+ [ ] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/master/CODE_REVIEW_GUIDELINES.md)_

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+  Java:
+    name: Java
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        module:
+          - app
+          - Backpack
```
Comment (georgegillams) [inline .github/workflows/ci.yml:63]: Smort! 👌

File: scripts/check-no-changes.sh [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```
Comment (georgegillams) [inline scripts/check-no-changes.sh:1]: Is this needed for GitHub actions to work? `#!/bin/bash` is more conventional 🤔

File: scripts/check-pristine-state [hunk 3]
```diff
@@ -1,58 +0,0 @@
...
-    .filter(isNotInList(filesToIgnore))
-    .filter(exists);
-
-  if (changedFiles.length) {
-    const changedFilesLog = changedFiles.join(',');
-    const ignoredFilesLog = filesToIgnore.join(',');
-    console.log(
-      `😰  Git shows some files have been changed. Changed files: ${changedFilesLog}. (ignoring ${ignoredFilesLog})`,
-    );
-
```
Comment (georgegillams) [inline scripts/check-pristine-state:48]: Listing the actual files that have changed is probably useful to someone debugging their failed CI run. Any way we could keep this behaviour?

**Review by georgegillams — COMMENTED** [review]
Just a couple of questions

File: scripts/check-pristine-state (hunk 3 above)
Comment (bvitaliyg) [reply scripts/check-pristine-state:48]: It's still there, see check-no-changes.sh :)

**Review by georgegillams — APPROVED** [review]

**Pull request merged**